server/requirements.txt

```diff
@@ -128,8 +128,9 @@ pyzmq==24.0.0
 qtconsole==5.4.0
 QtPy==2.3.0
 redis==4.3.4
-regex==2020.11.13
+regex==2022.1.18
```

Comment:
Is there a real reason to move to the latest regex?  I don't like to advance versions if we don't have a specific reason to do so.  It makes it more difficult for users if we keep upping the version as newest versions are not always available on on platforms.

Reply:
There was a dependency conflict with the pip package which we added. We can't remember which one causes it.